pkgs/development/compilers/dotnet/build-dotnet.nix

```diff
@@ -194,19 +194,27 @@ mkWrapper type (
       {
         inherit icu hasILCompiler;
       }
-      // lib.optionalAttrs (type == "sdk") {
-        packages = commonPackages ++ hostPackages.${hostRid} ++ targetPackages.${targetRid};
-        inherit targetPackages runtime aspnetcore;
-
-        updateScript =
-          let
-            majorVersion = lib.concatStringsSep "." (lib.take 2 (lib.splitVersion version));
-          in
-          [
-            ./update.sh
-            majorVersion
-          ];
-      };
+      // lib.optionalAttrs (type == "sdk") (
+        let
+          overridePackage = builtins.seq finalAttrs.finalPackage.drvPath;
```

Comment:
The name is confusing because it doesn't override anything AFAICT. It forces eval of the SDK in order to abort early when it's insecure, so how about `forceSDKEval`?

I also think that's not immediately obvious why this is done and warrants a code comment. Just put your commit message here as a comment.

(As a rule of thumb: Explain your changes in commit messages but explain your code in comments. If code needs to be explained regardless of the circumstances of the current change, that explanation should be a comment.)

Reply:
Yeah, good call on both points.  'override' came from a previous attempt which did actually override the nuget packages.

I've addressed this in latest.